psc-ide.el

```diff
@@ -241,6 +241,29 @@ Defaults to \"output/\" and should only be changed with
   (interactive)
   (psc-ide-send-sync psc-ide-command-quit))
 
+(defun psc-ide-server-start-with-psc-package-sources (root)
+  "start psc-ide-server with psc-package sources run in the root directory"
+  (interactive (list (read-directory-name "Project root?" (psc-ide-suggest-project-dir))))
+  (when (get-buffer "*psc-ide-server*") (kill-buffer "*psc-ide-server*"))
+  (cd root)
```

Comment:
I don't think changing the working directory of the Emacs process we're currently in is acceptable, so we'll need to figure out how to run the `psc-package` process in a different directory.

Reply:
Oh after reading some more I might be wrong. `cd` actually just sets `default-directory` for the current buffer, so we might be good.